config/flat.yaml

```diff
@@ -0,0 +1,80 @@
+# This is an example of how to make a flat-field image with imsim.
+
+
+# Load the imsim modules.
+modules:
+    - imsim
+
+input:
+
+    tree_rings:
+        # This enables TreeRingCenter and TreeRungFunc, which are stored as a dict here based
+        # on the detector name, so the right value can be accessed for each object.
+        # This file lives in the imsim repo.
+        file_name: "tree_ring_data/tree_ring_parameters_2018-04-26.txt"
```

Comment:
When I run using this config, I need to omit the `tree_ring_data/` folder from the `file_name`.

Reply:
Yep.  Fixed.  (This changed at some point on another branch that has since been merged.)